language/documentation/book/src/uses.md

```diff
@@ -7,27 +7,27 @@ The `use` syntax can be used to create aliases to members in other modules. `use
 There are several different syntax cases for `use`. Starting with the most simple, we have the following for creating aliases to other modules
 
 ```move
-use <address>::<module name>;
-use <address>::<module name> as <module alias name>;
+use <named_address|address>::<module name>;
+use <named_address|address>::<module name> as <module alias name>;
```

Comment:
Do we want to do this? I think we should just say `<address>` an `<address>` is defined (or should be defined elsewhere) as either: an ident, a decimal number, or a hex number

Reply:
Good point. I don't think we want to or need to do this. I'll update to `<address>`